{"uuid": "d2b90bbb-b8c3-4318-8b46-bfb1754134e2", "children": ["c65f6af8-29d8-4581-972a-23074723cd12", "6847366e-c92f-4408-b960-ca17cec650f6"], "befores": [{"name": "xunit_setup_class_fixture_TestUserRegister", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'test_user_data.xlsx'\n", "trace": "  File \"C:\\soft\\anaconda3\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 811, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\soft\\pycharm\\pythonProject\\test2\\request\\test_ydy\\test_register.py\", line 15, in setup_class\n    self.data_list = excel_to_list('test_user_data.xlsx', 'TestUserRegister')\n  File \"D:\\soft\\pycharm\\pythonProject\\test2\\request\\read_excel.py\", line 6, in excel_to_list\n    wb = xlrd.open_workbook(data_file)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\xlrd\\__init__.py\", line 111, in open_workbook\n    with open(filename, \"rb\") as f:\n"}, "start": 1615182386485, "stop": 1615182386485}], "start": 1615182386485, "stop": 1615182386507}